synapsis/src/test/java/com/soen/synapsis/unit/appuser/registration/RegistrationControllerTest.java

```diff
@@ -39,13 +45,15 @@ void tearDown() throws Exception {
     }
 
     @Test
+    @Order(1)
```

Comment:
It's not wrong, but is there a reason for specifying order here?

Reply:
It is related to what we discussed earlier about user authentication. Most tests will not pass unless I make sendValidAdminRegisterInfo() the last test to be executed. If you know of a better way, do let me know.

Reply:
In the future, you can probably just extract this to a separate class, or even make it an integration test, since that might be more appropriate, but it works for now.

Reply:
Ok.